Jenkinsfile

```diff
@@ -0,0 +1,29 @@
+#! groovy
+library 'pipeline-library'
+
+def publishableBranches = ['master']
+
+node('node && npm && npm-publish && nsp && retirejs') {
+  stage('Checkout') {
+    checkout scm
+    packageVersion = jsonParse(readFile('package.json'))['version']
+    currentBuild.displayName = '#${packageVersion}-${currentBuild.number}'
```

Comment:
You have to use double quotes for string interpolation in groovy, otherwise it won't insert the values you want here and will have that text set literally.